buzzmobile/tests/simulation/test_noop.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+class TestNoop(unittest.TestCase):
+    """Class showing proof of concept of simulation/unit test split.
```

Comment:
nit: add a TODO somewhere here to remove this test when we have simulation tests working. Feel free to assign it to me.